include/pybind11/pytypes.h

```diff
@@ -428,7 +460,8 @@ class PYBIND11_EXPORT_EXCEPTION error_already_set : public std::runtime_error {
     const object &trace() const { return m_trace; }
 
 private:
-    object m_type, m_value, m_trace;
+    mutable std::string m_lazy_what{};
```

Comment:
Do we need the trailing `{}`? (Seems unusual for a `std::string` member.)

Reply:
I was just debugging here.